src/main/java/Message.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Random;
+
+public class Message {
+    public static final String DELIMITER = "\n>====> >====> >====> >====> >====> >====> >====> >====> >====>\n";
+
+    private static final String LOGO_DEFAULT = "              _                 \n"
+            + "     /\\      | |                \n"
+            + "    /  \\   __| | __ _ _ __ ___  \n"
+            + "   / /\\ \\ / _` |/ _` | '_ ` _ \\ \n"
+            + "  / ____ \\ (_| | (_| | | | | | |\n"
+            + " /_/    \\_\\__,_|\\__,_|_| |_| |_|\n";
+
+    private static final String LOGO_BOLD = " █████╗ ██████╗  █████╗ ███╗   ███╗\n"
+            + "██╔══██╗██╔══██╗██╔══██╗████╗ ████║\n"
+            + "███████║██║  ██║███████║██╔████╔██║\n"
+            + "██╔══██║██║  ██║██╔══██║██║╚██╔╝██║\n"
+            + "██║  ██║██████╔╝██║  ██║██║ ╚═╝ ██║\n"
+            + "╚═╝  ╚═╝╚═════╝ ╚═╝  ╚═╝╚═╝     ╚═╝\n";
+
+    private static final String LOGO_3D = " ________  ________  ________  _____ ______      \n"
+            + "|\\   __  \\|\\   ___ \\|\\   __  \\|\\   _ \\  _   \\    \n"
+            + "\\ \\  \\|\\  \\ \\  \\_|\\ \\ \\  \\|\\  \\ \\  \\\\\\__\\ \\  \\   \n"
+            + " \\ \\   __  \\ \\  \\ \\\\ \\ \\   __  \\ \\  \\\\|__| \\  \\  \n"
+            + "  \\ \\  \\ \\  \\ \\  \\_\\\\ \\ \\  \\ \\  \\ \\  \\    \\ \\  \\ \n"
+            + "   \\ \\__\\ \\__\\ \\_______\\ \\__\\ \\__\\ \\__\\    \\ \\__\\\n"
+            + "    \\|__|\\|__|\\|_______|\\|__|\\|__|\\|__|     \\|__|\n";
+
```

Comment:
Perhaps the lineSeparator method could be used here? I believe it was mentioned that the behaviour of \n depends on the platform. Please correct me if I'm mistaken